barretenberg/cpp/src/barretenberg/stdlib/recursion/honk/verifier/recursive_verifier_instance.hpp

```diff
@@ -105,7 +109,13 @@ template <IsRecursiveFlavor Flavor> class RecursiveVerifierInstance_ {
      */
     VerifierInstance get_value()
     {
-        VerifierInstance inst;
+        auto inst_verification_key = std::make_shared<NativeVerificationKey>(instance_size, public_input_size);
+        inst_verification_key->pcs_verification_key = verification_key->pcs_verification_key;
+        for (auto [vk, inst_vk] : zip_view(verification_key->get_all(), inst_verification_key->get_all())) {
+            inst_vk = vk.get_value();
+        }
+
+        VerifierInstance inst(inst_verification_key);
         inst.pub_inputs_offset = pub_inputs_offset;
         inst.public_input_size = public_input_size;
         inst.log_instance_size = log_instance_size;
```

Comment:
not sure if some of these are unnecessary or not

Reply:
Same